lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -246,9 +253,10 @@
       data-display-name="${_("Discussion Community TAs")}"
       data-info-text="
         ${_("Community TA's are members of the community whom you deem particularly "
-        "helpful on the discussion boards. They can edit or delete any post, clear misuse flags, "
-        "close and re-open threads, endorse responses, and see posts from all cohorts. "
-        "Their posts are marked 'Community TA'.")}"
+        "helpful on the discussion boards. They can edit or delete any post, clear "
+        "misuse flags, close and re-open threads, endorse responses, and see posts from "
+        all cohorts. Their posts are marked as 'Community TA'. You can only give course "
```

Comment:
@lamagnifica  I think this might be the line that's causing the test failures.... looks like a missing double quote.